DigitalLearningSolutions.Data.Tests/Services/CourseContentServiceTests.cs

```diff
@@ -28,12 +28,14 @@ public void Get_course_content_should_return_course()
             var result = courseContentService.GetCourseContent(customisationId);
 
             // Then
-            var expectedCourse = new CourseContent
-            {
-                Id = 1,
-                CustomisationName = "Standard",
-                ApplicationName = "Entry Level - Win XP, Office 2003/07 OLD"
-            };
+            var expectedCourse = new CourseContent(
+                1,
+                "Entry Level - Win XP, Office 2003/07 OLD",
+                "Standard",
+                "N/A",

```

Comment:
Have you found a course which doesn't have this set to N/A? Might be useful to use one which doesn't have that for the tests, and would be good to see what the format usually is? 🙂

Reply:
The general format is along the lines of "5h 32m", stored as a string. I've changed the test case in 4ec78e9 🙂